tests/test_comment_spell_check.py

```diff
@@ -33,14 +33,16 @@ def test_basic(self):
         """Basic test"""
         runresult = subprocess.run(
             [
-                "comment_spell_check",
+                "python",
```

Comment:
This should be `sys.executable` to ensure that its the same executable as the the one currently being run.